### PR TITLE
Add eof command

### PR DESCRIPTION
**Pull request by bynect**
Add a command to jump to the end of file

### CODE REVIEW

File: src/config_dialog.c [hunk 1]
```diff
@@ -100,11 +100,11 @@ static void syntax(char **words, unsigned int words_len) {
     syntaxHighlight();
 }
 
-static void read_only_cmd(char **words, unsigned int words_len) {
+static void read_only(char **words, unsigned int words_len) {
     if (words_len == 1) {
-        if (!strncmp(words[0], "1", 1))
+        if (strcasecmp(words[0], "TRUE") == 0 || strcmp(words[0], "1") == 0)
```
Comment (arthurbacci) [inline src/config_dialog.c]: `== 0` makes the lines longer than they need to be

Comment (arthurbacci) [reply src/config_dialog.c]: `!` will make the same effect of `== 0` in this case.

File: src/config_dialog.c [hunk 2]
```diff
@@ -120,21 +120,22 @@ static void find(char **words, unsigned int words_len) {
...
-        for (unsigned int at = from_cur ? cy : 0; at < num_lines && (at != num_lines); ++at) {
+        for (unsigned int at = from_cur ? cy : 0; at < num_lines; ++at) {
             if (lines[at].length >= len &&
                 (index = uchar32_sub(from_cur && at == cy ? &lines[at].data[cx] : lines[at].data, words[words_len - 1], lines[at].length, len)) >= 0
             ) {
-                cursor.y = at;
-                cursor.x = index + len + (from_cur && at == cy) * cx;
-                cursor_in_valid_position();
-                syntaxHighlight();
+                change_position(index + len + (from_cur && at == cy) * cx, at);
```
Comment (arthurbacci) [inline src/config_dialog.c:127]: Thanks, this function will be useful for a lot of things.

File: src/config_dialog.c [hunk 3]
```diff
@@ -149,8 +150,9 @@ struct {
     {"save-as"          , save_as           },
     {"manual"           , manual            },
     {"syntax"           , syntax            },
-    {"read-only"        , read_only_cmd     },
+    {"read-only"        , read_only         },
```
Comment (arthurbacci) [inline src/config_dialog.c:153]: Isn't there any problem having a variable with the same name of a function?

Comment (bynect) [reply src/config_dialog.c:153]: > Isn't there any problem having a variable with the same name of a function?

I removed the read_only global variable and put it in the struct BUFFER in a previous commit merged some days ago

Comment (arthurbacci) [reply src/config_dialog.c:153]: I forgot that.

**Review by arthurbacci — APPROVED** [review]
Perfect.

**Comment by arthurbacci** [comment]
Thanks for the Pull Request!

**Pull request merged**